build/tools/appveyor.bat

```diff
@@ -18,11 +18,12 @@ echo --- Tools versions:
 g++ --version | head -1
 mingw32-make --version | head -1
 ar --version | head -1
+path C:\MinGW\bin;
```

Comment:
I guess maybe it was just other stuff in PATH making me use `cmd`. Can we just pull `%path%` from the command above, and just go with that?